public/javascripts/app.js

```diff
@@ -90,11 +85,14 @@ angular.module('uforia',
                 url: "/login",
                 templateUrl: "views/authentication/login.html",
                 controller: 'loginCtrl'
-                })
+            })
             .state('user', {
                 url: "/user",
                 templateUrl: "views/userOverview.html",
-                controller: 'userOverviewCtrl'
+                controller: 'userOverviewCtrl',
+                resolve: {
+                    role: isAdmin
```

Comment:
I looked into it but a function with optional parameters is not possible. I also need to call the injected services in isAdmin.
It is possible to add multiple roles like this:
 resolve: {
                    role: isAdmin,
                    role1: isManager
}